src/Lifecycle.ts

```diff
@@ -406,13 +406,42 @@ export function attemptTokenLogin(
         });
 }
 
+/**
+ * Load the pickle key inside the credentials or create it if it does not exist for this device.
+ * @param credentials Holds the device to load/store the pickle key, result is stored in credentials.pickleKey
+ */
+async function loadOrCreatePickleKey(credentials: IMatrixClientCreds): Promise<void> {
+    // Try to load the pickle key
+    const userId: string = credentials.userId;
+    const deviceId: string | undefined = credentials.deviceId;
+    let pickleKey: string | undefined = (await PlatformPeg.get()?.getPickleKey(userId, deviceId ?? "")) ?? undefined;
```

Comment:
Please avoid explicit typing explicitly, the types are fine implicit. https://github.com/element-hq/element-web/blob/develop/code_style.md only asks for explicit types where assignment does not happen on the same line